### PR TITLE
fix indentation of code

### PR DESCRIPTION
**Pull request by dev-aniketj**
fixing many indentation in the source code and code must be easy to read and understand.
thanks

### CODE REVIEW

**Comment by alankilborn** [comment]
Reformatting PR's are not accepted -- see contributing guidelines for the project.
Also, the code you changed is not Notepad++ code, it's just an archival copy, and copied from the Scintilla project.  Maybe Scintilla would welcome such a submission, but I doubt it.

**Comment by rdipardo** [comment]
> [. . .] the <del>Scintilla</del> **Lexilla** project

https://github.com/ScintillaOrg/lexilla

File: lexilla/include/SciLexer.h [hunk 1]
```diff
@@ -167,102 +165,101 @@
...
-#define SCE_USER_KWLIST_TOTAL                       28  // must always be <= KEYWORDSET_MAX
+#define SCE_UDL_VERSION_MAJOR 2
+#define SCE_UDL_VERSION_MINOR 1
+#define SCE_UDL_VERSION_BUILD 0
+#define SCE_UDL_VERSION_REVISION 12
 
-#define SCE_USER_TOTAL_DELIMITERS           8
-#define SCE_USER_TOTAL_KEYWORD_GROUPS       8
-#define SCE_USER_TOTAL_FOLDERS              6
+#define SCE_USER_KWLIST_COMMENTS 0
```
Comment (rdipardo) [inline lexilla/include/SciLexer.h:173]: This file is auto-generated by a Python script: https://github.com/ScintillaOrg/lexilla/blob/fa7721140059a3e16630893e3f5a6721f13063f7/scripts/LexFacer.py#L26

You should completely drop these changes if you decided to pitch this PR upstream: https://github.com/ScintillaOrg/lexilla

**Comment by alankilborn** [comment]
>Maybe ~Scintilla~ **Scintilla Org.** (?) would welcome such a submission, but I doubt it.

It gets awkward when an organization has one "product" and it is named the same as the organization.  THEN the org. decides to make other products.

**Comment by donho** [comment]
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests

**Comment by dev-aniketj** [comment]
Thanks, next time, I will do the right things.

**Pull request closed without merging**